lib/rex/crypto.rb

```diff
@@ -0,0 +1,14 @@
+module Rex::Crypto
+
+  def self.encrypt_aes256(iv, key, value)
```

Comment:
Probably worth copying the docs for this method here and marking with `@deprecated`

Reply:
@dwelch-r7 Looks like these are still missing the `@deprecated` metadata